CHANGELOG.md

```diff
@@ -6,11 +6,141 @@ document.
 
 ## Unreleased / In Rust Nightly
 
-[3ae8faf...master](https://github.com/rust-lang/rust-clippy/compare/3ae8faf...master)
+[4c41a22...master](https://github.com/rust-lang/rust-clippy/compare/3ae8faf...master)
+
+## Rust 1.55
+
+Current beta, release 2021-09-09
+
+[3ae8faf...4c41a22](https://github.com/rust-lang/rust-clippy/compare/3ae8faf...4c41a22)
+
+### New Lints
+
+* [`rc_mutex`]
+  [#7316](https://github.com/rust-lang/rust-clippy/pull/7316)
+* [`nonstandard_macro_braces`]
+  [#7299](https://github.com/rust-lang/rust-clippy/pull/7299)
+* [`strlen_on_c_strings`]
+  [#7243](https://github.com/rust-lang/rust-clippy/pull/7243)
+* [`self_named_constructor`]
+  [#7403](https://github.com/rust-lang/rust-clippy/pull/7403)
+* [`disallowed_script_idents`]
+  [#7400](https://github.com/rust-lang/rust-clippy/pull/7400)
+* [`disallowed_type`]
+  [#7315](https://github.com/rust-lang/rust-clippy/pull/7315)
+* [`missing_enforced_import_renames`]
+  [#7300](https://github.com/rust-lang/rust-clippy/pull/7300)
+* [`append_instead_of_extend`]
+  [#7270](https://github.com/rust-lang/rust-clippy/pull/7270)
+
+### Moves and Deprecations
+
+* Moved [`from_iter_instead_of_collect`] to `pedantic`
+  [#7375](https://github.com/rust-lang/rust-clippy/pull/7375)
+* Added `suspicious` as a new lint group for *code that is most likely wrong or useless*
+  [#7350](https://github.com/rust-lang/rust-clippy/pull/7350)
+  * Moved [`blanket_clippy_restriction_lints`] to `suspicious`
```

Comment:
I would write those as 

> Moved [`blanket_clippy_restriction_lints`] from `<lint_group>`

You can get the old lint groups from the PR body of #7350 

No need to mention here, that they were moved TO `suspicious` since this is already stated one line above.

Reply:
I wrote them this way to always have the same format for moved lints regardless of the context and I don't think that the original linting group is that interesting to the end user. If someone is really interested they can checkout the linked PR IMO. The main interest to me would actually be if the lint level has changed but most users don't directly know what group has which lint level. :thinking:

Reply:
Yeah makes sense. Then never mind me :)